src/System.Data.Common/src/System/Data/Common/DbConnectionStringBuilder.cs

```diff
@@ -266,7 +266,7 @@ private string ObjectToString(object keyword)
             }
             catch (InvalidCastException)
             {
-                throw new ArgumentException(nameof(keyword), "not a string");
+                throw new ArgumentException("not a string", nameof(keyword));
```

Comment:
I'm a little surprised this isn't a resource, but decided to leave as-is for now.